new-site/src/components/Image.scss

```diff
@@ -0,0 +1,13 @@
+@import '~hds-design-tokens/lib/all.scss';
+
+.image-container {
```

Comment:
We seem to have kebab-case and camelCase naming conventions mixed in the new repo for css styles. I wonder if we should use only one of them in the future for consistency?

Reply:
I guess we are using camelCase convention because of style imports in React package. It is easier to import `style.camelCase` that `style["kebab-case"]`